mkdocs_git_committers_plugin_2/plugin.py

```diff
@@ -118,13 +118,13 @@ def get_contributors_to_file(self, path, submodule_repo=None):
                             authors.append({'login': commit['author']['login'],
                                             'name': commit['author']['login'],
                                             'url': commit['author']['html_url'],
-                                            'avatar': commit['author']['avatar_url'] if user['avatar_url'] is not None else ''
+                                            'avatar': commit['author']['avatar_url'] if commit['author']['avatar_url'] is not None else ''
                                             })
                         if commit['committer'] and commit['committer']['login'] and commit['committer']['login'] not in [author['login'] for author in authors]:
                             authors.append({'login': commit['committer']['login'],
                                             'name': commit['committer']['login'],
                                             'url': commit['committer']['html_url'],
-                                            'avatar': commit['committer']['avatar_url'] if user['avatar_url'] is not None else ''
+                                            'avatar': commit['committer']['avatar_url'] if commit['author']['avatar_url'] is not None else ''
```

Comment:
Should reference the committer here, not the author.
```suggestion
                                            'avatar': commit['committer']['avatar_url'] if commit['committer']['avatar_url'] is not None else ''
```

Although, a cleaner option may just be
```py
'avatar': commit['committer']['avatar_url'] or ''
```